Automattic/wp-calypso
# Replace FEATURE_LIVE_CHAT_SUPPORT with FEATURE_PRIORITY_24_7_SUPPORT

Related to pau2Xa-5Yj-p2

## Proposed Changes

* Replaces `FEATURE_LIVE_CHAT_SUPPORT` with `FEATURE_PRIORITY_24_7_SUPPORT` and removes the `FEATURE_LIVE_CHAT_SUPPORT` feature.

## Why are these changes being made?


* pau2Xa-5Yj-p2

## Testing Instructions



This is a direct feature replacement, so there are no changes in functionality. Few instances that can be tested:
* Go to `/setup/site-setup/?siteSlug=<siteSlug>`.
* Continue until you land on the Design Picker.
* Select a Premium/WooCommerce/Partner theme, and click the Unlock Theme button.
* Confirm that there are no references to live chat.

<img width="876" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/a00b6dc9-76a5-4a86-ba9c-98327bac5630">
<img width="818" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/7e20c188-314b-4313-93f8-7f35f61a44d0">
<img width="848" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/055573ea-18b1-4f96-b72b-36f3be9674a7">


## Pre-merge Checklist
